beeline/middleware/django/__init__.py

```diff
@@ -53,6 +53,7 @@ def __call__(self, execute, sql, params, many, context):
                 "db.query": sql,
                 "db.query_args": params,
             })
+            beeline.add_rollup_field("db.query_count", 1)
```

Comment:
I can't comment a few lines later where it's relevant, but would it be a good idea to also add a roll-up field for the `db.duration`?

Reply:
There is [some prior art](https://github.com/honeycombio/beeline-go/blob/v1.4.1/wrappers/common/common.go#L309-L310) for both of these fields—count and duration—in the Go Beeline. If you update these field names to the Go version's (`db.call_count` and `db.duration_ms`) for consistency, I think we can get this in.

Reply:
The existing duration field for the individual span `db.duration` is seconds. Might need a different name for the rollup to avoid confusion. 🤔

Reply:
```suggestion
            beeline.add_rollup_field("db.call_count", 1)
```